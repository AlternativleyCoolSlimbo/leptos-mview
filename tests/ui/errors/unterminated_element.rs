use leptos_mview::mview;

fn main() {
    mview! {
        input type="text"
        "hi"
    };
}
