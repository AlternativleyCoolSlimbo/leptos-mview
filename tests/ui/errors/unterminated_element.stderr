error: unknown attribute
 --> tests/ui/errors/unterminated_element.rs:6:9
  |
6 |         "hi"
  |         ^^^^

error: child elements not found
 --> tests/ui/errors/unterminated_element.rs:5:9
  |
5 | /         input type="text"
6 | |         "hi"
  | |____________^
  |
  = note: if you don't want any child elements, end the element with a semi-colon `;` or empty braces `{}`.
